packages/adminrouter/extra/src/test-harness/tests/test_generic.config.yml

```diff
@@ -204,6 +206,7 @@ endpoint_tests:
   - tests:
       are_upstream_req_headers_ok:
         enabled: true
+        skip_authcookie_filtering_test: false
```

Comment:
During the design phase of the syntax for generic tests YAML, I was considering having "silent defaults", but eventually decided against it and this is a general rule for all YAML-based tests. The reason is that we want/need people unfamiliar with AR repo to contribute. Being explicit about what is enabled and what is not is, I think, helpful as they do not need to browse through docs and/or wonder why their copy-pasted test does not work. 

In this particular case, I suppose, it is even better as it directly states the fact that auth-cookie passing test is performed unless explicitly disabled.

This is not written in stone, esp. as the number of generic tests increases and the amount of extra information grows. I do not think this is the moment though.

Happy to hear your thoughts on that.